.gitignore

```diff
@@ -1,4 +1,4 @@
-/.composer.lock
+.composer.lock
```

Comment:
Why ?

Reply:
Well, this is wrong, and still wrong. I need to fix it. (the `.` is wrong)